app/javascript/src/components/Home/Hero.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Button } from "neetoui";
+
+export default function Hero() {
+  const features = [
+    "Uses Rails, React, Tailwind CSS and Webpacker.",
+    "Uses Devise, Honeybadger, Sidekiq, PostgreSQL, ActiveAdmin.",
+    "Heroku ready. Push to Heroku and it will work.",
+    "Uses slim for cleaner syntax over erb and better performance over haml.",
+    "Intercepts all outgoing emails in non production environment using gem mail_interceptor.",
+    "Uses SemaphoreCI for continuous testing.",
+    "Content compression via Rack::Deflater.",
+    "Auto-formats Ruby code with rubocop.",
+    "Auto-formats JavaScript and CSS code with prettier.",
+    "Performs background job processing 'inline' for heroku env. It means heroku can deliver emails.",
+    "Letter opener gem for development.",
+  ];
+
+  const featureLists = features.map((feature, index) => (
+    <li key={index}>{feature}</li>
+  ));
```

Comment:
Try to avoid using functions inside a component to render something. Use components to do it.

Reply:
Sure. I will make the change.

@vinay0x 
Just to clarify, what are the caveats associated with using a `map` function within a component to create jsx? Should I make it a general practice to extract out variables whose values are static and are composed of jsx elements?

Reply:
There are no caveats in using `map` within a component. My point was, don't have a method that renders something. Use another component. 

Either do 
```javascript
 features.map((feature, index) => (
    <Feature feature={feature}/>
  ));
```

or do 
```javascript
    <FeaturesList features={features}/>
```
and loop inside that component.